site-new/.gitignore

```diff
@@ -0,0 +1,20 @@
+# Dependencies
+/node_modules
```

Comment:
We manage the ignore patterns in the root `.gitignore` file:
https://github.com/line/armeria/blob/53d7cad0b572e8b0f587a59f587b68887e72dd77/.gitignore#L108

Reply:
Moved to the root `.gitignore` file!
Hope I don't forget to update it again when we change the directory name for deployment.